### PR TITLE
README Added

### PR DESCRIPTION
**Pull request by thetaxas2468**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
...
+
+	Idea 
+the idea of the project is to let group of people to organize together and buy straight from the supplier at lower cost.
+
+
+	Goals 
+the goals of the project are:
+ 1.to create simple and easy communication between an organized group of people and a supplier.
+2. lower the prices of food cost
+Plan our plan is to create a website that will communicate between the group of people and the supplier, removing the supermarkets from the equation, which will lead to lower the price for the customer.‏
```
Comment (brochwerger) [inline README.md:13]: Isn't the first word "Plan" a section title like "Goals" and "Idea". ?

The description of the plan is not correct (you just repeat the goals). The plan should be about HOW you intend to achieve the goals

**Review by brochwerger — COMMENTED** [review]
Please correct the English, in particular use proper capitalization

**Comment by thetaxas2468** [comment]
> Please correct the English, in particular use proper capitalization

Updated the read me ,is it good?

**Comment by yftacherzog** [comment]
@thetaxas2468 please squash the commits.

**Comment by thetaxas2468** [comment]
> @thetaxas2468 please squash the commits.

@yftacherzog  Done

**Comment by yftacherzog** [comment]
> > @thetaxas2468 please squash the commits.
> 
> @yftacherzog Done

Same comment from the other PRs about the commit message body.

File: README.md [hunk 2]
```diff
@@ -1 +1,20 @@
...
+---
+
+## **Idea**<br>
+The idea of the project is to let group of people to organize together and buy straight from the supplier at lower costs.
+
+---
+## **Goals** 
+Project Goals are:<br>
+	1. Create a simple and easy communication between an organized group of people and a supplier.<br>
+	2. Lower the prices of food cost
```
Comment (yftacherzog) [inline README.md:12]: Remove the leading spaces.

File: README.md [hunk 3]
```diff
@@ -1 +1,20 @@
...
+
+---
+## **Plan** 
+Our plan is to work by the division we have created between us and to make a beatiful Front-End page with decent design that has the ability to communicate with
+the backend that is connected with the database and after all that done it gives the clients the ablility to buy straight from the suppliers which mean the lower
+the prices of the food will cost and also with easy communication betwen clients and suppliers.
```
Comment (yftacherzog) [inline README.md:18]: I suggest running a spell check to catch some typos.

**Review by pazMenachem — COMMENTED** [review]
look at the suggestions.

File: README.md [hunk 4]
```diff
@@ -1 +1,20 @@
-# beyond08-group3
\ No newline at end of file
+# **buyTogether**
```
[the remark this answers is not in the document]
Comment (thetaxas2468) [reply README.md]: Done

**Review by LiorSilberman — COMMENTED** [review]
LGTM !

**Pull request merged**